### ISSUE TITLE
[4.0] Accessibility plugin should get a installation message

### ISSUE THREAD

**Issue by bembelimen** [joomla/joomla-cms#31792]
As a11y is very important for Joomla! 4, we should add an installation message to the backend.

Optional: add the function to enable the plugin with one click.

**Comment by brianteeman** [comment]
I disagree. This plugin is for edge cases and not for everyone. It doesn't improve accessibility or make something that is not accessible magically accessible.

**Comment by bembelimen** [comment]
Ok, fair enough.

**Comment by brianteeman** [comment]
If we were to add a post install message for this plugin then we should add one for every new feature in joomla 4.

That makes no sense

**Comment by pinochico** [comment]
It is not important to message after installation, but it is important that Joomla 4 has implemented accessibility rules at least in version A, better in version AA, as wordpress already has.
Many organizations would rather choose a wordpress that has accessibility rules implemented than pay developers to create a website on Joomla and then add those rules separately.

Unfortunately, for 2 years here:
https://github.com/joomla/accessibility
nothing happens.

**Comment by brianteeman** [comment]
@pinochico You obviously have not tested Joomla 4 as in almost every place it already meets WCAG 2.1 AA

> Unfortunately, for 2 years here:
https://github.com/joomla/accessibility
nothing happens.

If you wait for someone else to do something for you then nothing will ever happen

**Comment by brianteeman** [comment]
@bembelimen this comment is exactly why the plugin should not get a message as both you and @pinochico are mistaken in believing that without it Joomla is not accessible when the reality is that the admin of joomla 4 is perhaps the most accessible of any CMS.

_We really need to get better publicity when webaim.org survey showed joomla to be the best open sosurce cms for accessibility and we still get false comments about accessibility. (I refuse to comment about any other specific cms but it is well documented to not be accessible despite their pr._

**Comment by pinochico** [comment]
No need to create quarrels.

Maybe we don't understand (my google translate isn't mostly ok):
By accessibility, I mean meeting the rules by
https://www.w3.org/TR/WCAG21/


We can say that Joomla meets them, but now in Joomla4 I am editing the view for the basic Joomla modules menu and search, so that the output meets these rules.

Maybe I'm doing something wrong ..

**Comment by chmst** [comment]
>in Joomla4 I am editing the view for the basic Joomla modules menu and search, so that the output meets these rules.

In J4, the modules meet the rules for  WCAG 2.1 AA. Can you give more information what you are doing and why?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31792">issues.joomla.org/tracker/joomla-cms/31792</a>.</sub>

**Comment by chmst** [comment]
@pinochico we are always open and grateful for every contribution to a11y. But please open new issues if you find any violation of rules. It is not in scope of this one. 

@bembelimen I agree with @brianteeman that for this plugin no extra message is needed.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31792">issues.joomla.org/tracker/joomla-cms/31792</a>.</sub>

**Comment by wilsonge** [comment]
Closing this as not required. But definitely would appreciate if you could raise issues for any violations you are finding @pinochico !

**Comment by pinochico** [comment]
I will create new issue, sure, but men, now its holiday :)

**Comment by wilsonge** [comment]
Enjoy the holidays! Thankyou I’m advance whenever you get time to submit them!
